docs/source/ufs_utils.rst

```diff
@@ -665,3 +665,57 @@ Run script
 ----------
 
 To run, use the machine-dependent script under ./util/weight_gen
+
+***************************************************
+UFS_UTILS utilities
+***************************************************
+
+gdas_init
+=========
+
+Introduction
+------------
+
+The gdas_init utility is used to create coldstart initial conditions for global cycled experiments using the chgres_cube program.  It has two components: one that pulls the input data required by chgres_cube from HPSS, and one that runs chgres_cube. The utility is only supported on machines with access to HPSS:
+
+     * Hera
+     * Jet
+     * WCOSS2
+     * S4 (Only the chgres_cube step is supported, not the data pull step.)
+
+Location
+--------
+
+Find it here: ./util/gdas_init
+
+Build UFS_UTILS and set 'fixed' directories
+-------------------------------------------
+
+     * Invoke the build script from the root directory: ``./build_all.sh``
+     * Set the 'fixed' directories using the script in the 'fix' subdirectory: ``./link_fixdirs.sh emc $MACHINE`` (where MACHINE is 'hera', 'jet', 'wcoss2', or 's4')
```

Comment:
Totally fine with how you have your commands here but you could also use the `::` code block if you want. See example here from g-w RTD documentation: https://github.com/KateFriedman-NOAA/global-workflow/blob/develop/docs/source/setup.rst?plain=1#L28:L32

Reply:
Thanks for the suggestion. How does this look?

https://georgegayno-noaaufs-utils.readthedocs.io/en/gi_doc/ufs_utils.html#gdas-init

Reply:
Looks great!